src/Symfony/Component/Validator/Resources/translations/validators.fr.xlf

```diff
@@ -402,6 +402,14 @@
                 <source>The value of the netmask should be between {{ min }} and {{ max }}.</source>
                 <target>La valeur du masque de réseau doit être comprise entre {{ min }} et {{ max }}.</target>
             </trans-unit>
+            <trans-unit id="104">
+                <source>The filename is too long. It should have {{ filename_max_length }} character or less.|The filename is too long. It should have {{ filename_max_length }} characters or less.</source>
+                <target>Le nom du fichier est trop long. Il doit contenir au maximum {{ filename_max_length }} caractère.|Le nom de fichier est trop long. Il doit contenir au maximum {{ filename_max_length }} caractères.</target>
+            </trans-unit>
+            <trans-unit id="105">
+                <source>The password strength is too low. Please use a stronger password.</source>
+                <target>Le mot de passe est trop fragile. Veuillez utiliser un mot de passe plus robuste.</target>
```

Comment:
```suggestion
                <target>Le mot de passe est trop faible. Veuillez utiliser un mot de passe plus robuste.</target>
```

faible <-> robuste
fragile <-> solide

wdyt?

Reply:
I took `robuste` from https://fr.wikipedia.org/wiki/Robustesse_d%27un_mot_de_passe and `fragile` from https://www.larousse.fr/dictionnaires/synonymes/robuste/18548 antonyms.

I don’t really like `faible` because it has a broad meaning, but it’s just my opinion!

Reply:
copy that, thx for sharing :)

Reply:
Qui de ça? Ça me semble classique comme terminologie.
La force du mot de passe est trop faible. Veuillez utiliser un mot de passe plus fort.